SPO/README.md

```diff
@@ -0,0 +1,102 @@
+# How to create a SPFx extension solution with AppInsights installed from scratch
```

Comment:
@DawgFan hi Matt, can you please help check the language and pictures in README? Thanks! 😁

Reply:
Thank you for adding all the great screen shots!